rkt/config/config.go

```diff
@@ -15,66 +15,17 @@
 package config
 
 import (
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
 	"net/http"
-	"os"
-	"path/filepath"
-	"sort"
-	"strings"
 
 	"github.com/coreos/rkt/common"
-	"github.com/hashicorp/errwrap"
+	baseconfig "github.com/coreos/rkt/pkg/config"
```

Comment:
As above, `config` seems to be fine here since it does not conflict with any other imported pkg.